content/news/2024/04/2024-04-09-meet-your-digital-experience-council.md

```diff
@@ -0,0 +1,58 @@
+---
+source: ""
+date: 2024-04-09
+title: Meet your Digital Experience Council
+deck: Your partners in delivering a digital-first public experience
+summary: The Digital Experience Council plays a critical role in coordinating
+  governmentwide efforts and assisting agencies in delivering digital
+  experiences that meet the public’s needs and expectations.
+# Controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+weight: 1
+author: Jonathan Finch
```

Comment:
See this `authors` [front matter](https://github.com/GSA/digitalgov.gov/blob/4652eaf723dfe272fd05bde9d522617ec493bc83/content/news/2022/11/2022-11-08-jargon-madness-a-plain-language-exercise.md?plain=1#L11) to properly set the author.

Should read:
```
authors:
  - jonathan-finch
``` 

This is causing the build issue.